src/driver/compile.ml

```diff
@@ -16,6 +16,7 @@ type compiled = {
   include_dirs : Fpath.Set.t;
   impl : impl option;
   pkg_args : pkg_args;
+  current_package : string;
```

Comment:
This change has already been made in various flavor in two other PRs: [here](https://github.com/ocaml/odoc/pull/1084/files#diff-aab3f7bebea0819961f37ab5de1d58bef4863e48e77cd55d26b914a3b63ef87dR19) and [there](https://github.com/ocaml/odoc/pull/1145/files#diff-aab3f7bebea0819961f37ab5de1d58bef4863e48e77cd55d26b914a3b63ef87dR19)!

We'll see which one is kept during the rebase.